[Collaboration] Latest doc changes (#381)
ezsystems/user-documentation: docs/content_management/collaborative_editing.md

Then, in a pop-up window, you can invite users to newly created session.

You can only invite users with permissions to access the shared content.
Users without permissions are grayed out and you can't select them and invite them to the session.

You can invite two types of users:

> - **Internal** - by searching their name or email address. These users can either edit the content item or preview it, depending on your choice. Depending on their account permissions, they might be able to invite other users.

Reviewer: To avoid double "depending".
```suggestion
- **Internal** - by searching their name or email address. These users can either edit the content item or preview it, depending on your choice. Their account permissions determine whether they can invite additional users.
```

![Change access](img/change_access.png)

> Additionally, you can share a direct link to the collaborative session without specyfing a particular user.

Reviewer: ```suggestion
Additionally, you can share a direct link to the collaborative session without specifying a particular user.
```

To do it, click the **Copy link** button.
Link is copied to the clipboard and you can share it with the users through communication channels.

> Use the **Link settings** button to control whether an account in the system is needed to preview content when using the direct link.

Reviewer: Sounds a bit clearer in my opinion:
```suggestion
Use the **Link settings** button to control whether a user needs an account in the system to preview content using the direct link.
```

Author: Definitely, thank you 😄

![Participants list](img/participants_list.png)

#### Real-time collaboration Terms of Service changes

Real-Time Collaboration service is only available after accepting its Terms and Conditions.

> When Ibexa releases a new version of this document, you will see a notification in content edit forms that the new version must be accepted before the appointed deadline.

Reviewer: As suggested by vale:
```suggestion
When [[= product_name_base =]] releases a new version of this document, you will see a notification in content edit forms that the new version must be accepted before the appointed deadline.
```
